### ISSUE TITLE
Preparing images to run core integration tests

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud-docker/testing#4]
I have made some changes to the owncloud-docker/server image and now I can run integration tests there. https://github.com/SergioBertolinSG/server/commits/master

The key part is to use the qa tarball including the tests.

It is required to build the image using that variable:

OWNCLOUD_TARBALL=https://download.owncloud.org/community/owncloud-daily-master-qa.tar.bz2

To run the tests there are some extra changes needed, adding some permissions and have data folder linked to the mounted volume for data.

It would be nice to have that image built in the registry able to be downloaded.

**Comment by tboerger** [comment]
The base image already works fine for that, I have already explained that on our chat.

**Comment by SergioBertolinSG** [comment]
Not really, it needs some little changes and also to build the image with some parameters:

```
docker build -t owncloudserver:latest --build-arg OWNCLOUD_TARBALL=http://download.owncloud.org/community/owncloud-daily-master-qa.tar.bz2 
--build-arg LDAP_TARBALL=https://github.com/owncloud/user_ldap/archive/master.zip --no-cache .
```

and in the docker-compose file https://github.com/owncloud-docker/server/blob/master/docker-compose.yml#L5

The image pulled from the registry wasn't build with those parameters, it would be required to have an image build this way to just do the docker-compose up and have it all set up.

**Comment by tboerger** [comment]
You are always referring to the owncloud/server image build. Just use owncloud/base with a volume mounting the source of the qa tarball and it just works. That's exactly what jenkins is doing.

**Comment by SergioBertolinSG** [comment]
Ok, I'll try that way and see. Where is that jenkins configuration located?
